Protocols/IEC61850/SWDevice-010805/EventNotification.md

```diff
@@ -0,0 +1,162 @@
+## EventNotification messages
+
+### Description
+
+Buffered report sent from device to platform containing information about 1 event.
```

Comment:
Provide some information about the content of an event, which is created in OSGP (data to the platform triggered by the event).

Reply:
Done.